docs/supports-and-hinges/structuralpointsupport.md

```diff
@@ -12,7 +12,13 @@ StructuralPointSupport represents a support of the analytical model in a node. T
 |              Name             |      String      |                                                                                                       Sn6                                                                                                      |                                           yes                                           | Human readable unique name of the support                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 |              Type             |       Enum       |                                                                    <p>Fixed</p><p></p><p>Hinged</p><p></p><p>Sliding</p><p></p><p>Custom</p>                                                                   |                                            no                                           | Type of constraint support in general. Has just informative value. Actual boundary conditions are set per transition/rotation per direction.                                                                                                                                                                                                                                                                                                                                 |
-|              Node             |      String      |                                                                                                       N5                                                                                                       |                                           yes                                           | The name of the node to which is point support related                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| Boundary condition | Enum | In node<p>On beam | yes | Specifies to which type of object the support is assigned |
+| Node | String | N5 | yes, if Boundary condition = In node | The name of the node to which is point support related  |
+| 1D member | String | B1 | yes, if Boundary condition = On beam | The name of the [StructuralCurveMember](../structural-analysis-elements/structuralcurvemember.md) to which the support is assigned |
```

Comment:
Not sure if this should just be "Member" instead of "1D member"
If I look through the rest of the docs, we either use "Member" or "Reference member"...

Reply:
Good point, I changed it to Member